pkg/exporter/sumologicexporter/README.md

```diff
@@ -8,6 +8,10 @@ The following configuration options are supported:
 - `compress_encoding` (optional): Compression encoding format, either empty string (`""`), `gzip` or `deflate` (default `gzip`).
 Empty string means no compression
 - `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default `1_048_576` (1MB) is used.
+- `transform_metadata` (optional): Specifies whether metadata attributes should be transformed
```

Comment:
Now that I read it again, should I add a `Metadata Transformation` subsection to the README that will list *all* the metadata that is being transformed? Effectively copying the `metadataTransformation` variable from [transform_metadata.go:5](https://github.com/SumoLogic/sumologic-otel-collector/pull/14/files#diff-09f7a032d741100d7ab9a0d2ed0a5e226b7c36dc411bb01de71cac48ba1d1eefR5)

Reply:
I think it would be good to add a table with those transformations in README

Reply:
Added section on metadata transformation to README, with a list of all transformed attribute keys.